README.md

```diff
@@ -1,2 +1,5 @@
 # LearnJavaScriptBuilding7Games
 Learn JavaScript building 7 games
+
+Based on the instructive video of Ania Kubow.
```

Comment:
Please, include the name of the training company.

Reply:
Done!